config.yml

```diff
@@ -0,0 +1,7 @@
+api_version: 1.1.0
+stac_version: 0.9.0
+url: http://www.brockmann-consult.de/xcube-geoDB-openEO
+id: xcube-geodb-openeo
+title: xcube geoDB Server, openEO API
+description: Catalog of geoDB collections.
+datastore-class: xcube_geodb_openeo.core.GeoDBDataStore
```

Comment:
If this refers to a data store whose interface is defined by OpenEO, please add "OpenEo" into its name, because xcube also defines a _DataStore_ within its data store framework.

Reply:
This datastore is not defined by OpenEO, it is an implementation detail: I introduced a `GeoDBDataStore` for production, and a `MockDataStore` for testing. Maybe we should rename those to `...DataSource` to differentiate from the xcube DataStore?